library/src/main/java/io/ona/kujaku/utils/LocationSettingsHelper.java

```diff
@@ -0,0 +1,71 @@
+package io.ona.kujaku.utils;
+
+import android.app.Activity;
+import android.content.IntentSender;
+import android.util.Log;
+
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.common.api.PendingResult;
+import com.google.android.gms.common.api.ResultCallback;
+import com.google.android.gms.common.api.Status;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationServices;
+import com.google.android.gms.location.LocationSettingsRequest;
+import com.google.android.gms.location.LocationSettingsResult;
+import com.google.android.gms.location.LocationSettingsStatusCodes;
+
+/**
+ * Created by Ephraim Kigamba - ekigamba@ona.io on 31/10/2018
+ */
+
+public class LocationSettingsHelper {
+
+    private static final String TAG = LocationSettingsHelper.class.getName();
+
+    public static void checkLocationEnabled(Activity activity) {
```

Comment:
Does this check whether Location Services are enabled or does it check that specific Location Services parameters are satisfied?

It's not clear from the name, so maybe some short documentation on the method may be helpful.

Or maybe it could be renamed to `verifyLocationServicesSpecs` as a suggestion?